03-Custom-Actions-In-Python/01-What-are-Actions.ipynb

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
<p>Under "Why Python Custom Actions?" - instead of listing a number of things that we considered as developers, list some concrete things that users can do with them. You can keep the same 4 topics, just reword it to make declarative statements of what a user will be able to do and avoid making any judgements about C++ vs Python or users expertise.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>+1 to this! I added a similar comment on the index page but it works better here for more specific examples.</p>

Reply:
<p>I find reviewNB a bit clumsy and accidentally added it to the branch instead of the PR, idk if my comment will show up so here it is:</p><blockquote>Could add an example of a concrete benefit here: "For example, if you want to change the particle size during a simulation, you would have to stop the simulation, make the change, and run it again. Now, you can set up a custom action to avoid having to restart the simulation"</blockquote>

Reply:
<p>I am unable to make inline suggestions:</p><ul><li>what is "act"?  <strong>Change? Modify?</strong> Seems a bit odd to define a word with itself.</li><li>What is a hyperparameter? (or link to it if defined in prereqs)</li></ul>

Reply:
<p>I also give +1 to concrete examples! It could be good to hint here at what examples will follow.</p>

Reply:
<p>New user perspective: I could use a little more explanation of how updaters and tuners are different. I understand that the updater changes the state (could remind users what the state is) and the tuner doesn't.</p><p>But that the tuner "modif(ies) another object's hyperparameters." doesn't help me understand without a little more info.</p><p> </p>